tests/LanguageServerCompletion/Extension/TestExtension.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Phpactor\Extension\LanguageServerCompletion\Tests\Extension;
+
+use Phpactor\Container\Container;
+use Phpactor\Container\ContainerBuilder;
+use Phpactor\Container\Extension;
+use Phpactor\FilePathResolver\Expander\ValueExpander;
+use Phpactor\FilePathResolverExtension\FilePathResolverExtension;
+use Phpactor\MapResolver\Resolver;
+
+class TestExtension implements Extension
+{
+    public function load(ContainerBuilder $container): void
+    {
+        $this->registerFilePathExpanders($container);
+    }
+
+    public function configure(Resolver $schema): void
+    {
+    }
+
+    private function registerFilePathExpanders(ContainerBuilder $container): void
+    {
+        $container->register('core.file_path_resolver.project_config_expander', function (Container $container) {
```

Comment:
This one comes from the phpactor main project. I put this here for a minimal setup.

Reply:
hmm, yes this is annoying.